src/app/frontend/secretdetail/detail.html

```diff
@@ -15,17 +15,34 @@
 -->
 
 <div layout="column">
-    <kd-secret-info secret="::$ctrl.secretDetail"></kd-secret-info>
-    <kd-info-card>
-        <kd-info-card-section name="[[Data | Secrets info details section data.]]">
-            <kd-info-card-entry ng-repeat="(key, value) in ::$ctrl.secretDetail.data"
-                                title="{{::key}}">
-                <pre ng-if="::!$ctrl.secretDetail.isHiddenType" class="kd-pre-block">{{::$ctrl.formatDataValue(value)}}</pre>
-                <kd-toggle-hidden-text  ng-if="::$ctrl.secretDetail.isHiddenType"
-                                        placeholder="{{::value.length}} [[bytes | Secrets info details section bytes.]]"
-                                        text="{{ ::value }}">
-                </kd-toggle-hidden-text>
-            </kd-info-card-entry>
-        </kd-info-card-section>
-    </kd-info-card>
+  <kd-secret-info secret="::$ctrl.secretDetail"></kd-secret-info>
+  <kd-info-card>
+    <kd-info-card-header>[[Data | Secrets info details section data.]]</kd-info-card-header>
+    <kd-info-card-section>
+      <div layout="row" class="kd-info-card-entry kd-secret-detail-row" ng-repeat="(key, value) in ::$ctrl.secretDetail.data">
+        <div flex="nogrow" style="position: relative">
+          <md-button ng-click="isActive = !isActive" class="md-icon-button">
+            <!-- SVG drawing to simulate crossed-eye icon.
+                 This should be removed if possible. -->
+            <svg ng-if="isActive" width="40px" height="40px" viewBox="0 0 40 40"
```

Comment:
Sadly, we do not use this library :( We use https://material.io/icons/, which is limited in scope. There is no eye-off icon there. 

We could try evaluating and including other material design libraries, but this requires much effort of integration, validation and license eveluation. Can we do this in a separate task? I can spawn an issue.